4-fleetscope/modules/env_baseline/variables.tf

```diff
@@ -24,14 +24,9 @@ variable "fleet_project_id" {
   type        = string
 }
 
-variable "scope_id" {
-  description = "The fleet scope ID"
-  type        = string
-}
-
-variable "namespace_id" {
-  description = "The fleet namespace ID"
-  type        = string
+variable "scope_ids" {
+  description = "The fleet scope IDs"
+  type        = list(string)
```

Comment:
Can we use `namespace_ids` as the variable here instead? It makes it clearer that we are creating the namespaces at this stage, and creating a scope for each namespace

Reply:
sure, rename to namespace_ids